subprojects/core/src/integTest/groovy/org/gradle/api/tasks/IncrementalTasksIntegrationTest.groovy

```diff
@@ -94,6 +98,9 @@ class IncrementalTasksIntegrationTest extends AbstractIntegrationSpec {
         def touchOutputs() {
         }
 
+        def createOutputs() {
```

Comment:
This would be better if the method name included something about it being called only in the non-incremental case.